.changeset/silver-doors-shout.md

```diff
@@ -0,0 +1,68 @@
+---
+"@wso2is/admin.identity-verification-providers.v1": patch
```

Comment:
We can only patch bump the core, console and administrators modules and changeset will take care of the rest :)

Reply:
fixed with https://github.com/wso2/identity-apps/pull/6700/commits/2f9ead44743ee603856ee9d29924c6595497540a